### PR TITLE
do not add polygons with way_pixels == 0

### PR DESCRIPTION
**Pull request by farfromrefug**
As discussed in slack this is a test PR to see the benefit / cost of filtering way_pixels == 0

### CODE REVIEW

**Comment by farfromrefug** [comment]
Running this PR on France i saw a 2.7% size decrease in the mbtiles. Also this should mean less polygons to draw for the client => faster

File: layers/landcover/landcover.sql [hunk 1]
```diff
@@ -259,115 +283,138 @@ CREATE OR REPLACE FUNCTION layer_landcover(bbox geometry, zoom_level int)
...
          SELECT geometry, 
-                subclass
+                subclass,
+                area
          FROM osm_landcover_polygon
          WHERE zoom_level >= 14
            AND geometry && bbox
-     ) AS zoom_levels;
+         WHERE zoom_level >= 14
+           AND geometry && bbox
```
Comment (ZeLonewolf) [inline layers/landcover/landcover.sql:415]: Copy/paste error: lines 414-415 are a duplicate of 412-413

Comment (farfromrefug) [reply layers/landcover/landcover.sql]: @ZeLonewolf thanks!

File: layers/landcover/landcover.sql [hunk 2]
```diff
@@ -259,115 +283,136 @@ CREATE OR REPLACE FUNCTION layer_landcover(bbox geometry, zoom_level int)
             )
 AS
 $$
+SELECT geometry, class, subclass
+FROM (
 SELECT geometry,
        landcover_class(subclass) AS class,
-       subclass
+        subclass,
+        case when (subclass not in ('park')) then nullif(name,'') end as name,
```
Comment (ZeLonewolf) [inline layers/landcover/landcover.sql:291]: There is no `name` column available here.

Comment (farfromrefug) [reply layers/landcover/landcover.sql:291]: Thanks something went wrong while i created that branch. Name exists in my fork

**Comment by farfromrefug** [comment]
All good now. I have tested it locally and it build find (finally). Could we have the bot trst it to see if we gain anything?

**Comment by TomPohys** [comment]
Thanks @farfromrefug, on the tests areas the size was cut by 6.4%, That awesome! Will you do any more changes or it can be merged?

**Comment by TomPohys** [comment]
Let's switch to `draft` again to be clear, there is some work in progress. Anyway, thank you for working on this!

**Comment by TomPohys** [comment]
Hi @farfromrefug, did the local tests decrease the tile size?

**Comment by farfromrefug** [comment]
@TomPohys Did not have much time to work on this honestly.
I ll close this and it can be revisited if necessary.
Sorry

**Pull request closed without merging**